lametro/templates/partials/search_header.html

```diff
@@ -1,3 +1,8 @@
 <p class="h4">
     Search Metro Board Reports
-</p>
\ No newline at end of file
+</p>
+<p>
+  <small>
```

Comment:
<img width="1299" alt="Screen Shot 2019-09-19 at 1 35 32 PM" src="https://user-images.githubusercontent.com/12176173/65271231-5da3f100-dae2-11e9-90db-486a4d1317be.png">

this is actually quite large on my screen. what do you think about a smaller font size?